src/xo-app/vm/tab-disks.js

```diff
@@ -50,6 +51,117 @@ import {
   subscribeResourceSets,
 } from 'xo'
 
+const COLUMNS_VM_PV = [
+  {
+    itemRenderer: vdi => (
+      <Text
+        value={vdi.name_label}
+        onChange={value => editVdi(vdi, { name_label: value })}
```

Comment:
Editing the VDI via editable inputs doesn't work probably due to conflicts with [keyboard shortcuts](https://github.com/vatesfr/xo-web/blob/master/src/common/sorted-table/index.js#L731-L736) when hitting Enter.